tests/test_doctestplus.py

```diff
@@ -314,7 +314,7 @@ def test_normalize_with_ellipsis(self):
         c = OutputChecker()
         got = []
         for char in ['A', 'B', 'C', 'D', 'E']:
-            got.append('%s %s' % (char, float(ord(char))))
+            got.append('{} {}'.format(char, float(ord(char))))
```

Comment:
Hmm why no f-string here?

Reply:
Automated mess I guess 🤷‍♀️

I mean, I pushed stuff before boarding a flight, but as I said above there is some weird stuff happening locally already so I'll very likely have to follow up with more manual fixes

Reply:
No rush on my end. Thanks!

Reply:
local issues were due to outdate plugins